src/hoc/withCustomWallpaper.jsx

```diff
@@ -0,0 +1,35 @@
+import useCustomWallpaper from 'hooks/useCustomWallpaper'
+import React from 'react'
+import { useClient } from 'cozy-client'
+
+/**
+ * @function
+ * @description HOC to provide custom wallpaper from context as prop
+ *
+ * @param  {Component} WrappedComponent - wrapped component
+ * @returns {Function} - Component that will receive wallpaper as prop
+ */
+const withCustomWallpaper = WrappedComponent => {
+  const Wrapped = props => {
+    const client = useClient()
+    const {
+      fetchStatus,
+      data: { wallpaperLink }
+    } = useCustomWallpaper(client)
+    // @ts-ignore
```

Comment:
ts-ignore? Do we have TS here?

Reply:
I just let it as it was. Now it is removed